include/boost/geometry/algorithms/crosses.hpp

```diff
@@ -30,7 +30,8 @@
 
 #include <boost/geometry/geometries/concepts/check.hpp>
 
-#include <boost/geometry/algorithms/detail/relate/relate.hpp>
+#include <boost/geometry/algorithms/relate.hpp>
+#include <boost/geometry/algorithms/detail/relate/relate_impl.hpp>
```

Comment:
Do we use the suffix _impl more often?
There is also an impl folder in projections, so it is not the only usage of this abbreviation
But is there not an alternative? Thought I saw already some correspondence about this going by but cannot find it in this PR